src/zimscraperlib/inputs.py

```diff
@@ -42,7 +47,14 @@ def handle_user_provided_file(
 
     if str(source).startswith("http"):
         logger.debug(f"download {source} -> {dest}")
-        stream_file(url=str(source), fpath=dest)
+        if user_agent:
+            if USER_AGENT_RE.match(user_agent):
+                headers = {"User-Agent": user_agent}
+            else:
+                raise ValueError(f"Invalid User-Agent format: {user_agent}")
```

Comment:
I don't think we need to enforce the user-agent policy, this could bite us at some point.

@rgaudin WDYT?

Reply:
We definitely don't want that and even if we wanted that regex is way to precise/complex for this.